src/runtime/components/Calendar.vue

```diff
@@ -88,6 +88,10 @@ export interface CalendarProps<R extends boolean = false, M extends boolean = fa
   monthControls?: boolean
   /** Show year controls */
   yearControls?: boolean
+  /**
+   * @defaultValue 'solid'
+   */
+  variant?: 'solid' | 'outline' | 'soft' | 'subtle' | 'ghost'
```

Comment:
The `variant` should be dynamically typed based on the theme:
```suggestion
  variant?: Calendar['variants']['variant']
```